### PR TITLE
add some additional messaging when a boot command fails to type

### PR DESCRIPTION
**Pull request by jhawk28**
trying to get some more information for #8957

### CODE REVIEW

File: builder/vsphere/common/step_boot_command.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func (s *StepBootCommand) Run(ctx context.Context, state multistep.StateBag) mul
 			Shift:    keyShift,
 		})
 		if err != nil {
-			return fmt.Errorf("error typing a boot command: %v", err)
+			return fmt.Errorf("error typing a boot command (code, down) `%d, %t`: %w", code, down, err)
```
Comment (sylviamoss) [inline builder/vsphere/common/step_boot_command.go:98]: Does `%w` work here? It didn't work for me in this example: https://play.golang.org/p/FkJkN4JFui9

Comment (jhawk28) [reply builder/vsphere/common/step_boot_command.go:98]: %w was added in 1.13 as a way to wrap errors. It works with the fmt.Errorf(), not the fmt.Printf()

Comment (sylviamoss) [reply builder/vsphere/common/step_boot_command.go:98]: oh! Sorry, my eyes 👀  did pay attention to the `Errorf` detail!

Comment (jhawk28) [reply builder/vsphere/common/step_boot_command.go:98]: Its not ingrained in my memory yet. I have to look it up each time still.

**Pull request merged**